### ISSUE TITLE
READMEs for crates are not consistant with code doc comments

### ISSUE THREAD

**Issue by nuke-web3** [paritytech/substrate#10089]
Example of where divergence occurs:
1. README.md: [# Balances Module](https://github.com/paritytech/substrate/tree/12b6441ef138303f50fa2fb20895fbf57c8e4735/frame/balances#balances-module)
2. Doc comments: https://github.com/paritytech/substrate/blob/12b6441ef138303f50fa2fb20895fbf57c8e4735/frame/balances/src/lib.rs#L18

I was under the impression that these `README.md`s were _automatically_ generated by some CI. This is not the case, or at least it is not working as intended. I would propose we _remove all README.md_ files here that are associated with code comments, and emphasize in the top level README that all docs can be viewed in a nice rendered format for many versions of Substrate (including the approximate HEAD of master) here: 
### https://docs.substrate.io/rustdocs/

**Comment by thiolliere** [comment]
yeah I thought they were automatically generated but only before releasing, thus the version on master is often outdated.

I also agree to remove them and use the same README with good links everywhere
